### PR TITLE
Pin 133

### PR DESCRIPTION
**Pull request by decode9**

### CODE REVIEW

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (rotcax) [inline .vscode/settings.json]: Please ignore this file

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -9,37 +9,36 @@ import Toast from '../Toast'
 import styles from './styles.module.scss'
 
 const Layout = ({ children }) => {
-  const {
-    showFooter,
-    modal: { contact }
-  } = useSelector((state: any) => state.intermitence)
-
+  const { showFooter, modal: { contact } } = useSelector((state: any) => state.intermitence)
+  const [form, setForm] = useState({ name: '', lastname: '', email: '', message: '' })
```
Comment (rotcax) [inline src/components/Layout/index.tsx]: Use formik for handle form information

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Comment by the-code-workers** [comment]
Your Code is succesfully Success

**Review by rotcax — APPROVED** [review]

**Pull request merged**